partitioner/src/test/java/io/confluent/connect/storage/partitioner/TimeBasedPartitionerTest.java

```diff
@@ -583,6 +580,37 @@ public void testRecordFieldTimeStringExtractor() {
     validateEncodedPartition(encodedPartition);
   }
 
+  @Test
+  public void testRecordFieldTimeAsMillisStringExtractorRecord() {
+    long millis = DATE_TIME.getMillis();
+    String timeStr = String.valueOf(millis);
```

Comment:
as below

Reply:
It's used below in two places, prefer to keep it as is.